frontend/src/types.ts

```diff
@@ -523,14 +523,14 @@ export type TagContent = {
 
 export type ImageTagInfo = {
   name: string;
-  content: TagContent;
+  content?: TagContent;
```

Comment:
According to the backend, this is always present.
```suggestion
  content: TagContent;
```